test/test_compare256.cc

```diff
@@ -65,15 +65,16 @@ TEST_COMPARE256(c, compare256_c, 1)
 TEST_COMPARE256(native, native_compare256, 1)
 #else
 
-#if defined(UNALIGNED_OK) && BYTE_ORDER == LITTLE_ENDIAN
+#if BYTE_ORDER == LITTLE_ENDIAN && OPTIMAL_CMP >= 32
```

Comment:
I don't think you need to care about `OPTIMAL_CMP` for unit tests. These should just all be run.